### PR TITLE
Fix issue 16141 - Organizations page should have a menu item

### PR DESCRIPTION
**Pull request by wilzbach**
Thanks @qznc for pointing this out. We have to be a bit careful about not adding too much to the menu, but I recently purged to items from the menu, so I guess we are good.
Moreover I took the liberty to sort this alphabetically (it's easier to parse for the human eye).

Btw there are many other great pages that are currently hard to find, see e.g. http://dlang.org/sitemap.html

### CODE REVIEW

File: dlang.org.ddoc [hunk 1]
```diff
@@ -226,8 +226,9 @@ $(SUBMENU
     $(ROOT_DIR)bugstats.php, Bug Tracker,
     https://dlang.org/blog, Blog,
     https://forum.dlang.org, Forums,
+    https://github.com/dlang, GitHub,
     irc://irc.freenode.net/d, IRC,
-    https://github.com/dlang, D on GitHub,
+    $(ROOT_DIR)/orgs-using-d.html, Orgs using D,
     https://wiki.dlang.org, Wiki,
     https://twitter.com/search?q=%23dlang, Twitter,
```
Comment (aG0aep6G) [inline dlang.org.ddoc]: > Moreover I took the liberty to sort this alphabetically

T comes before W in the alphabet.

**Comment by wilzbach** [comment]
> T comes before W in the alphabet.

Oh I am quite ashamed I overlooked that. Thanks!

File: dlang.org.ddoc [hunk 2]
```diff
@@ -226,10 +226,11 @@ $(SUBMENU
     $(ROOT_DIR)bugstats.php, Bug Tracker,
     https://dlang.org/blog, Blog,
     https://forum.dlang.org, Forums,
+    https://github.com/dlang, GitHub,
     irc://irc.freenode.net/d, IRC,
-    https://github.com/dlang, D on GitHub,
-    https://wiki.dlang.org, Wiki,
+    $(ROOT_DIR)/orgs-using-d.html, Orgs using D,
```
Comment (CyberShadow) [inline dlang.org.ddoc]: No `/` after `ROOT_DIR`...

**Comment by wilzbach** [comment]
> No / after ROOT_DIR...

Seems like one of those mistakes I tend to make. Hopefully I will remember that one :S

**Comment by CyberShadow** [comment]
As in #1359 I'm not sure alphabetical sorting brings much here. Many nitpicking but I think GitHub does not belong between Forums and IRC.

**Comment by wilzbach** [comment]
> As in #1359 I'm not sure alphabetical sorting brings much here. Many nitpicking but I think GitHub does not belong between Forums and IRC.

As said in the other PR I wanted to give vertical dividers a try - I am not entirely sure yet:

image

At least I know that with dividers we can't have single items:

image

**Comment by wilzbach** [comment]
@CyberShadow @aG0aep6G  how did you like the first screenshot with the white dividers?

**Comment by JackStouffer** [comment]
Style preference: these vertical dividers are a bit too in your face. Try #962A23 for its color

screen shot 2016-06-10 at 2 24 47 pm

**Comment by aG0aep6G** [comment]
> @CyberShadow @aG0aep6G how did you like the first screenshot with the white dividers?

I think white is too harsh. Alternative suggestions:

Just space:

seb-separators-space

The style from the narrow-layout menu:

seb-separators-groovy

**Comment by qznc** [comment]
"Just space" looks like a layout error. I prefer the dark lines.

**Comment by wilzbach** [comment]
> "Just space" looks like a layout error. I prefer the dark lines.

Rebased. Thanks for your feedback!!

**Comment by MartinNowak** [comment]
I think this broke our nightly builds (http://nightlies.dlang.org/).
[Issue 16167 – chm-nav.json generation is broken](https://issues.dlang.org/show_bug.cgi?id=16167)

**Pull request merged**